src/shared/components/ui/progress.tsx

```
@@ -0,0 +1,31 @@
+'use client';
+
+import * as React from 'react';
+import * as ProgressPrimitive from '@radix-ui/react-progress';
+
+import { cn } from '@/shared/libs/utils/cn';
```

ESLint

[error] Run autofix to sort these imports!

(simple-import-sort/imports)